boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -15,6 +15,9 @@ Bluetooth™, GPS, displays, and camera sensors. As with other i.MX processors,
 i.MX RT1050 also has rich audio and video features, including LCD display,
 basic 2D graphics, camera interface, SPDIF, and I2S audio interface.
 
+The following document refers to the discontinued MIMXRT1050-EVK board, for the
```

Comment:
change the comma to a period and start a new sentence:

     This document refers to the discontinued MIMRT1050-EVK board.  For the